mitodl/hugo-course-publisher
# update markdown with latest fixes for RES courses

#### Pre-Flight checklist


#### What are the relevant tickets?
https://trello.com/c/f1ufpmW7/44-missing-data-in-masterjson-for-supplemental-resources

#### What's this PR do?
Updates the sample markdown with the latest changes to support RES pages.

#### How should this be manually tested?
Run the site, verify that it loads properly and that you can see pages in RES courses.

#### Screenshots (if appropriate)
![Screen Shot 2020-07-01 at 4 57 24 PM](https://user-images.githubusercontent.com/12089658/86291088-7d615d00-bbbc-11ea-921f-2634a7efb185.png)
![Screen Shot 2020-07-01 at 5 00 08 PM](https://user-images.githubusercontent.com/12089658/86291100-83573e00-bbbc-11ea-8acf-88a06c61b59f.png)
